packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
Both approaches has pros and cons. sourceId in events and targetId in commands could just be id. And even though sourceId and targetId are more explicit then only id it got me confused at the beginning. I think it was Rhys who told me "targetId is the id". I explained the same thing to one of new users of Space recently.